app/src/main/java/com/instructure/template/projectCodeHere/GetCourses.java

```diff
@@ -0,0 +1,40 @@
+package com.instructure.template.projectCodeHere;
+
+import com.instructure.template.loginTemplate.api.ApiPrefs;
+import com.instructure.template.loginTemplate.api.models.Course;
+import retrofit2.Call;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+import retrofit2.http.GET;
+import retrofit2.http.Header;
+import retrofit2.http.Path;
+
+import java.util.List;
+
+public interface GetCourses {
+    @GET("/api/v1/users/{user_id}/courses")
+    Call<List<Course>> coursesCall(
+            @Path("user_id") Long user_id,
+            @Header("Authorization") String authorization,
+            @Header("User-Agent") String userAgent
+    );
+
+    public static final Retrofit retrofit = new Retrofit.Builder()
+            .baseUrl(ApiPrefs.getFullDomain())
+            .addConverterFactory(GsonConverterFactory.create())
+            .build();
+}
+    //This is how one would implement this interface (returns json data)
+    /*GetCourses getCourses = GetCourses.retrofit.create(GetCourses.class);
```

Comment:
Why is this stuff commented? You could probably throw a [UnsupportedOperationException](https://docs.oracle.com/javase/7/docs/api/java/lang/UnsupportedOperationException.html) instead.

Reply:
because you are in an interface. you cant run it there